.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,3 +11,8 @@ _[Describe the modifications you've done.]_
 ### Result:
 
 _[After your change, what will change.]_
+
+### Checklist:
+
+[] Code and documentation should follow the style of the surrounding code.
```

Comment:
Instead of just surrounding code, this can link to our [style guide](https://github.com/apple/swift-testing/blob/main/Documentation/StyleGuide.md)

Reply:
I forgot we had that 🙈 Done!